static/scss/program-page.scss

```diff
@@ -372,6 +372,7 @@
 
   .faqs {
     margin-top: 20px;
+    background-color: transparent;
```

Comment:
@arslanashraf7 is that intentional? It doesn't seem to show any changes, can you confirm that after removing it?

Reply:
Yes, It to keep the background transparent on FAQ tab of the program page.